setup.py

```diff
@@ -887,16 +888,24 @@ def configure_extension_build():
 
     # Cross-compile for M1
     if IS_DARWIN:
+        report('-- detected IS_DARWIN')
```

Comment:
This information is already printed in Cmake summary
```suggestion
```

Reply:
removed